src/components-examples/angular/search/search-header-autocomplete/search-header-autocomplete-example.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+import { FormControl } from '@angular/forms';
+
+/**
+ * @title Search Header Mode With Autocomplete
+ * @order 30
+ */
+@Component({
+  selector: 'sbb-search-header-autocomplete-example',
+  templateUrl: './search-header-autocomplete-example.html',
+})
+export class SearchHeaderAutocompleteExample {
+  searchValues: string[] = [];
+  searchControl = new FormControl('');
+
+  filteredOptions = options.slice(0);
+
+  handleSearch(value: string) {
+    this.searchValues = [value, ...this.searchValues].slice(0, 5);
```

Comment:
I think slicing is an overkill for the showcase examples and can only confuse consumers?

Reply:
I don't want the list to grow unlimited. Should I maybe add a comment to explain why this is done?